src/go/rpk/pkg/cli/cmd/cluster/config/config.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package config
+
+import (
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/cli/cmd/common"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+)
+
+func NewConfigCommand(fs afero.Fs) *cobra.Command {
+	var (
+		all            bool
+		adminURL       string
+		adminEnableTLS bool
+		adminCertFile  string
+		adminKeyFile   string
+		adminCAFile    string
+	)
+
+	command := &cobra.Command{
+		Use:   "config",
+		Short: "Interact with cluster configuration properties",
+		Long: `Interact with cluster configuration properties
+
+Cluster properties are redpanda settings which apply to all nodes in the cluster.
+These are separate to node properties, which are set with 'rpk redpanda config'.
+
+Use the 'edit' subcommand to interactively modify the cluster configuration, or
+'export' and 'import' to write configuration to a file that can be edited and
+read back later.
+
+Modified properties are propagated immediately to all nodes.  The 'status' subcommand
+can be used to verify that all nodes are up to date, and identify any settings
+which were rejected by a node, for example of a node is running a different
+redpanda version that does not recognize certain properties.
```

Comment:
Can this have a short paragraph on what low level tunables are? They're mentioned in edit/export, but it's not clear what they are. When / why do I need to worry about low level tunables?

Reply:
I've added an explanatory paragraph in the config.go long docstring (the --all option applies to all of edit, import, export)